src/App.js

```diff
@@ -1,8 +1,24 @@
 import './App.css';
+import {BrowserRouter as Router, Switch, Route,} from "react-router-dom";
+import Home from './pages/home/Home';
+import Subreddit from './pages/subreddit/Subreddit';
+
 
 function App() {
   return (
-    <h1>Reddit</h1>
+      <>
+      <Router>
+        <Switch>
+          <Route exact path="/">
+              <Home />
+          </Route>
+        <Route path="/subreddit/:subredditid">
+            <Subreddit />
+        </Route>
+        </Switch>
+      </Router>
+
+      </>
```

Comment:
De lege tags zijn niet nodig, omdat het maar één child bevat
```suggestion
      <Router>
        <Switch>
          <Route exact path="/">
              <Home />
          </Route>
        <Route path="/subreddit/:subredditid">
            <Subreddit />
        </Route>
        </Switch>
      </Router>
```